microsite/data/plugins/scaffolder-backend-roadie-aws.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Scaffolder AWS cli actions
+author: roadie.io
+authorUrl: https://roadie.io/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=scaffolder-aws
+category: Scaffolder
+description: Here you can find some AWS cli actions
+documentation: https://github.com/RoadieHQ/roadie-backstage-plugins/blob/main/plugins/scaffolder-actions/scaffolder-backend-module-aws/README.md
+iconUrl: img/scaffolder-aws.-logo.png
```

Comment:
Could you perhaps find a public source for this logo instead? We prefer to not host logos within the repo